portfolio/src/main/webapp/index.html

```diff
@@ -85,6 +85,36 @@
             </div>
           </div>
 
+          <div class="col-md-6">
+            <div class="card mb-4 shadow-sm bg-light">
+              <title>GetchaNotes</title>
+              <p class="text-dark mx-auto project-title">GetchaNotes</p>
+              <img class="border border-dark" src="images/getchanotes.png" width="100%" height="100%">
+              <rect width="100%" height="100%"/>
+              <div class="card-body">
+                <p class="card-text text-dark">GetchaNotes is a web-based application that converts images 
+                    of physical notes (hand-written or otherwise) to a digitized version. Notes are categorized 
+                    using OCR and NLP technologies and can be filtered by topic, viewed, and downloaded.</p>
+                <div class="d-flex justify-content-between align-items-center">
+                  <div class="btn-group">
+                    <a class="btn btn-sm btn-outline-secondary" href="https://summer20-sps-33.uc.r.appspot.com/" 
+                        target="_blank" role="button">View</a>
+                    <button class="btn btn-sm btn-outline-secondary" type="button" data-toggle="collapse" 
+                        data-target="#moreGetchaNotes" aria-expanded="false" aria-controls="collapseExample">
+                        More</button>
+                  </div>
+                </div>
+                <div class="collapse" id="moreGetchaNotes">
+                  <br>
+                  <p class="collapse-text text-dark">Team Members: Josh Chan, Joymaneet Kaur, Lily Ou, 
```

Comment:
Super optional since this is your portfolio, but you might consider linking to pages for each person? (Might be their own portfolios, or their LinkedIns or Twitters or whatever.)